driver/src/memory/xaie_mem.c

```diff
@@ -77,6 +77,8 @@ AieRC XAie_DataMemWrWord(XAie_DevInst *DevInst, XAie_LocType Loc,
 		return XAIE_INVALID_DATA_MEM_ADDR;
 	}
 
+	XAIE_DEBUG("Added to test cr-bot")
+
```

Comment:
This is an example yet again!

```suggestion
 return NULL;
```

Maybe this works